dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1072,7 +1072,7 @@ javaCmd()
     # We direct STDERR to /dev/null to avoid that noise.
     {
       #  Launch with stdin set to /dev/null to avoid file-handle complications on criu restore
-      "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
+      ${JAVA_CHECKPOINT_PREFIX_CMD}"${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
```

Comment:
Can you document the variable ${JAVA_CHECKPOINT_PREFIX_CMD}? At least in general that it's there for use by by container build scripts

Reply:
Done